circuit-retained/src/commonMain/kotlin/com/slack/circuit/retained/RetainedStateRegistry.kt

```diff
@@ -117,14 +117,7 @@ internal class RetainedStateRegistryImpl(retained: MutableMap<String, List<Any?>
         // the first provider returned null(nothing to save) and the second one returned
         // "1". When we will be restoring the first provider would restore null (it is the
         // same as to have nothing to restore) and the second one restore "1".
-        list
-          .map { it.invoke() }
-          .onEach { value ->
-            // Allow nested registries to also save
-            if (value is RetainedStateRegistry) {
```

Comment:
This never worked, as `value` is always a `RetainableHolder.Value` which needs to be unwrapped. Everything works without this code anyway, as `rememberRetained` will call `saveAll` for us.